tests/console/dracut.pm

```diff
@@ -20,13 +20,16 @@ use power_action_utils 'power_action';
 sub run {
     my ($self) = @_;
     $self->select_serial_terminal;
+    select_console("root-console") if check_var('BACKEND', 'svirt');
```

Comment:
even if you select root console on svirt, you are still selecting serial terminal on svirt one step ahead

Reply:
Since we don't have remote svirt-xen worker for testing, I was debugging the issue with serial-terminal in this case, please ignore this WIP PR! 
I think I found out why this is not working on svirt-xen and need to do other tests.